tutorials/ui/gui_using_fonts.rst

```diff
@@ -156,6 +156,60 @@ property that accepts a Font resource.
    The texture filter mode can also be set on individual nodes that inherit from CanvasItem
    by setting :ref:`CanvasItem.texture_filter <class_CanvasItem_property_texture_filter>`.
 
+Font outlines and shadows
+-------------------------
+
+Font outlines and shadows can be used to improve readability when the background
+color isn't known in advance. For instance, this is the case for HUD elements
+that are drawn over a 2D/3D scene.
+
+Font outlines are available in most nodes that derive from Control, in addition
```

Comment:
I just checked every control node in 4.1. The only two that don't have the option are GraphEdit (which has no font options at all) and GraphNode. I think it would make more sense to just say these are the only two without the option instead of "most nodes have it"

Reply:
I think having it as "most" is fine in lieu of possible future changes or additions, these details are hard to track and tend to get outdated. That said, I don't feel super strongly about it, so if you want to discuss this further, a PR to make just that change is welcome as a followup to have a discussion :)